### PR TITLE
Instant return

### PR DESCRIPTION
**Pull request by JulienMattiussi**
If the boss is already visited or already accessible, we don't need to browse the dungeon again

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -11,19 +11,13 @@ let startDungeon = (init = false, options = {}) => {
...
-        case "verbose":
-          verbose = options[key];
-          break;
-        default:
-          break;
-      }
-    });
+  if (options) {
+    if (options.chestMode) {
+      chestMode = options[key];
```
Comment (Nitix) [inline script.js]:
```suggestion
      chestMode = options.chestMode;
```

**Review by Nitix — CHANGES_REQUESTED** [review]

File: script.js [hunk 2]
```diff
@@ -209,6 +203,12 @@ let configureDungeon = () => {
       return false;
     }
     verbose && console.log("Go to boss");
+    const event = new MouseEvent("click");
+    const bossTile = document.querySelector(".tile-boss, .tile-ladder");
+    if (bossTile) {
+      bossTile.dispatchEvent(event);
+      DungeonRunner.handleInteraction();
```
Comment (Nitix) [inline script.js:210]: You should return true here too.

**Review by Nitix — COMMENTED** [review]
Does this work with seen but not visited ? ie with flash
Does this work with multi-layer dungeon ?

File: script.js (hunk 2 above)
Comment (JulienMattiussi) [reply script.js:210]: In fact, I shouldn't

The script will try to click on the tile every time, but sometimes, the click won't have any impact if the boss tile is still not clickable
in this case, we want the function to continue as if we had never clicked

Comment (JulienMattiussi) [reply script.js:210]: and yes, it should work with multi layer dongeon ;)

**Pull request merged**